jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebInfConfiguration.java

```diff
@@ -219,12 +216,13 @@ public void findAndFilterContainerPaths (final WebAppContext context)
             loader = loader.getParent();
         }
         
-        if (LOG.isDebugEnabled()) LOG.debug("Matching container urls {}", containerUris);
+        if (LOG.isDebugEnabled())
+            LOG.debug("Matching container urls {}", containerUris);
         containerPathNameMatcher.match(containerUris);
 
         //if running on jvm 9 or above, we we won't be able to look at the application classloader
         //to extract urls, so we need to examine the classpath instead.
-        if (JavaVersion.VERSION.getPlatform() >= 9)
+        if (targetPlatform >= 9)
```

Comment:
This is a change of logic.   The target platform for quickstart generation is different to the platform that we are currently running on.

Reply:
Good catch, I reverted it back.